src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -271,6 +277,67 @@ std::vector<std::vector<std::string>> LiveFunctionsDataView::GetContextMenuWithG
   return menu;
 }
 
+ErrorMessageOr<void> LiveFunctionsDataView::ExportAllEventsToCsv(
+    const std::filesystem::path& file_path, const std::vector<int>& item_indices) {
+  ErrorMessageOr<orbit_base::unique_fd> result = orbit_base::OpenFileForWriting(file_path);
+  if (result.has_error()) {
+    return ErrorMessage{absl::StrFormat("Failed to open \"%s\" file: %s", file_path.string(),
+                                        result.error().message())};
+  }
+  const orbit_base::unique_fd& fd = result.value();
+
+  // Write header line
+  constexpr const char* kFieldSeparator = ",";
+  constexpr const char* kLineSeparator = "\r\n";
+  constexpr size_t kNumColumns = 5;
+  const std::array<std::string, kNumColumns> kNames{"Name", "Thread", "Start", "End",
+                                                    "Duration (ns)"};
+  std::string header_line;
+  for (size_t i = 0; i < kNumColumns - 1; ++i) {
+    header_line.append(FormatValueForCsv(kNames[i]));
+    header_line.append(kFieldSeparator);
+  }
+  header_line.append(FormatValueForCsv(kNames[kNumColumns - 1]));
```

Comment:
This can be done nicely like this:
```
std::string header_line = absl::StrJoin(kNames, kFieldSeparator, [](std::string* out, const std::string& name){
out->append(FormatValueForCsv(name));
})
```

Reply:
Done! Thanks for the suggestion!